### ISSUE TITLE
Scheduling page has a11y issues that affect functionality

### ISSUE THREAD

**Issue by malini** [thunderbird/appointment#1040]
**Describe the bug**
From a user testing with the latest versions of NVDA and JAWS on Windows 11, as well as VoiceOver on the latest version of iOS 18:

When first opening the availability page as though I was going to book in a meeting/appointment with someone, the list of available times and dates does not always seem to appear, instead there is just the heading of the page ("[Users]'s Availability" in my case) and a “confirm selection” button. When this happens, a full refresh of the page (ctrl +F5) is required in order to bring the dates and times in. I'm not sure if this is an accessibility issue i.e. the schedule has appeared but the screen reader doesn't see it, or if it is a general bug, but thought I would mention it here just in case.

It is not clear to a screen reader user what date and time is being selected when navigating through the schedule. The available times are not represented as links or buttons, so a user might not realise they are clickable. Pressing enter on a timeslot does select that particular slot, but it is not always clear exactly what date and time has been selected.

It is not possible to navigate to a new month using a screen reader. If I wanted to book in a meeting with someone in June or July for example, I would have no way to do this as there isn't any option to move to the next month that is accessible using a screen reader.

**Comment by rtanglao** [comment]
This issue is based on Zendesk ticket 513 for those who have access.
For those who don't here's some more perhaps relevant details
>If you would like an example of a system that does this well, I would highly recommend taking a look at the schedule page of Calendly if possible. I have booked meetings with people using this service before and it provides an extremely accessible experience.

> I would love to use Thunderbird Appointment to assist me in scheduling appointments with others, but at the moment the current accessibility issues prevent me from doing so as I'm not willing to send out an inaccessible scheduling page. The dashboard and account settings pages could also do with some improvements, but they are certainly still usable and I'd say the scheduling page should be the priority at the moment.

 @laurelterlesky are there any other details you need from this person in Zendesk 513?

**Comment by rtanglao** [comment]
* Related to #299

**Comment by malini** [comment]
moving this to early bird so we can start chipping away at this

**Comment by laurelterlesky** [comment]
Rebecca is has some a11y documentation here: https://www.figma.com/design/GTgK7wRXQwRwncIpSEk6Sv/Web_2026_Q1_Appointment_EarlyBird_v01?node-id=2039-936&t=PggN9hTYS3NgsCpk-1

High level for the dashboard, but we'll need to dig deeper into the calendar specifically. Might need to have a meeting on this.

**Comment by solangevalverde** [comment]
Booking page a11y notes are now available on [Zeplin](https://zpl.io/LLBoLZJ) as well as in [figma](https://www.figma.com/design/GTgK7wRXQwRwncIpSEk6Sv/appt_web_2026-Q1_earlyBird_v001?node-id=3010-1972&t=h7mVfIHBiRQPFD9b-1). Dropping the notes here as well just in case but annotations will be found in Zeplin.

![Image](https://github.com/user-attachments/assets/69173d54-ec94-4261-915e-399111c19b2f)
